src/config.ts

```diff
@@ -8,8 +8,13 @@ export const getAppMountUri = () =>
 export const getApiUrl = () => window.__SALEOR_CONFIG__.API_URL;
 export const SW_INTERVAL = parseInt(process.env.SW_INTERVAL, 10);
 export const IS_CLOUD_INSTANCE = process.env.IS_CLOUD_INSTANCE === "true";
-export const MARKETPLACE_URL = process.env.MARKETPLACE_URL;
-export const SALEOR_APPS_ENDPOINT = process.env.SALEOR_APPS_ENDPOINT;
+export const MARKETPLACE_URL = window.__SALEOR_CONFIG__.MARKETPLACE_URL;
+export const MARKETPLACE_SALEOR_APPS_PAGE_PATH =
+  window.__SALEOR_CONFIG__.SALEOR_APPS_PAGE_PATH;
+export const MARKETPLACE_SALEOR_APPS_JSON_PATH =
+  window.__SALEOR_CONFIG__.SALEOR_APPS_JSON_PATH;
+export const MARKETPLACE_APP_TEMPLATE_GALLERY_PATH =
+  window.__SALEOR_CONFIG__.APP_TEMPLATE_GALLERY_PATH;
```

Comment:
**suggestion**:  how about creating functions just like we have with getApiUrl? (consistency)

Reply:
I think there should be 1 layer with raw ENVs, second layer with some functions to resolve them

However, Dashboard is not my scope so I suggest you plan conventions and refactors with the team. When I contribute, I follow style I see in the codebase